File: back_end/VehicleTracker/src/main/java/com/model/Driver.java
package com.model;


import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.*;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Driver {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int driverId;
    private String driverName;
    private String licenseNumber;
    private String email;
    private String password;
}

File: back_end_dotnet/vehicleTracker_dotnet/Model/VTModels/Driver.cs
using System.ComponentModel.DataAnnotations;

namespace VehicleTracker.Models;

public class Driver
{

    [Key]
    public int DriverId { get; set; }


    [Required]
    public string DriverName { get; set; } 


    [Required]
    public string LicenseNumber { get; set; }


    [Required]
    public string Email { get; set; } 


    [Required]
    public string Password { get; set; }


    public bool IsActive { get; set; } = false;


}
